examples/entry.js

```diff
@@ -0,0 +1,13 @@
+const Log = require('../src/log')
+const EntryIO = require('../src/entry-io')
+const Ipfs = require('ipfs')
+const { MemStore } = require('orbit-db-test-utils')
+const IdentityProvider = require('orbit-db-identity-provider')
+
+module.exports = {
+  Log,
+  EntryIO,
+  Ipfs,
+  MemStore,
+  IdentityProvider
+}
```

Comment:
What's the purpose of this example? There's not a lot to it, so perhaps add a comment describing what to look for, or if this is not intended as an example, move the file to a more appropriate place.

Reply:
This should still be remove, no?

Reply:
Negative. This is the entry point for the webpack example build, so we need that to build the bundle which runs the examples